dartagnan/src/main/antlr4/LitmusPTX.g4

```diff
@@ -151,7 +151,7 @@ fenceAlias
     ;
 
 barrier
-    :   Barrier Period CTA Period barrierMode value
+    :   Barrier Period CTA Period barrierMode constant
```

Comment:
You cannot make this change. PTX [allows](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#parallel-synchronization-and-communication-instructions-bar-barrier) to use registers (as long as they hold a value `0...15`) for the barrier id:
```
barrier{.cta}.sync{.aligned}      a{, b};
...
Operands a, b, and d have type .u32; operands p and c are predicates. Source operand a specifies a logical barrier resource as an immediate constant or **register** with value 0 through 15
```